installer/SystemConfiguration.php

```diff
@@ -144,45 +144,74 @@ public function createAdminUser($userName, $password) {
     }
 
     /**
-     * Set the instance identifier value
+     * Set the instance identifier value to db
      * @param $organizationName
      * @param $email
-     * @param $randomNumber
+     * @param $adminFirstName
+     * @param $adminLastName
+     * @param $host
+     * @param $country
+     * @param $ohrmVersion
      */
-    public function setInstanceIdentifier($organizationName, $email, $randomNumber) {
-        $instanceIdentifier = $organizationName . '_' . $email . '_' . date('Y-m-d') . $randomNumber;
+    public function setInstanceIdentifier($organizationName, $email, $adminFirstName, $adminLastName, $host, $country, $ohrmVersion) {
+        $instanceIdentifier = $this->createInstanceIdentifier($organizationName, $email, $adminFirstName, $adminLastName, $host, $country, $ohrmVersion);
         $query = "INSERT INTO `hs_hr_config` (`key`, `value`) VALUES (?, ?)";
         $dbConnection = $this->createDbConnection();
         $statement = $dbConnection->prepare($query);
-        $statement->execute(array(self::KEY_INSTANCE_IDENTIFIER, base64_encode($instanceIdentifier)));
+        $statement->execute(array(self::KEY_INSTANCE_IDENTIFIER, $instanceIdentifier));
     }
 
     /**
      * Create instance identifier value
      * @param $organizationName
-     * @param $organizationEmail
-     * @param $createdDate
-     * @param $randomNumber
+     * @param $email
+     * @param $adminFirstName
+     * @param $adminLastName
+     * @param $host
+     * @param $country
+     * @param $ohrmVersion
      * @return string
      */
-    public function createInstanceIdentifier($organizationName, $organizationEmail, $createdDate, $randomNumber) {
-        return base64_encode($organizationName . '_' . $organizationEmail . '_' . $createdDate . $randomNumber);
+    public function createInstanceIdentifier($organizationName, $email, $adminFirstName, $adminLastName, $host, $country, $ohrmVersion) {
+        if(is_null($host))
+        {
```

Comment:
This is against PSR2 coding standards 
https://www.php-fig.org/psr/psr-2/#5-control-structures